tests/mock_crust/churn.rs

```diff
@@ -74,20 +74,20 @@ impl ExpectedGets {
                        dst: Authority,
                        nodes: &[TestNode],
                        min_group_size: usize) {
-        let src_dest = src.to_destination();
-        let dst_dest = dst.to_destination();
+        let src_addr = src.to_destination();
+        let dst_addr = dst.to_destination();
```

Comment:
Let's not mix random cleanups with actual features/fixes/tests/etc.
As discussed, the `Destination` type will probably go away anyway, and then this confusion will be automatically removed, too, so I'd just not change this for now. If you must, change `src` and `dst` to `src_auth` and `dst_auth`, and make these two here just `src` and `dst`: We sometimes use "address" for a `XorName`, so these names would be just as confusing. (But as a separate commit, together with any other cleanups not related to adding the clients as senders and recipients.)